python/pyproject.toml

```
@@ -1,6 +1,6 @@
 [project]
 name = "ommx"
-version = "0.1.1"
+version = "0.1.2"
```

```suggestion
version = "0.2.0"
```

Rustと違ってPythonのルールはもう少し弱いはずですが、ややこしいのでこちらもあげてしまいましょう